### ISSUE TITLE
SmartCity-Accessibility Server Side #451 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#292]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] 70f9201a1ad0c7710375ef57e8743ad721c36b08 - put convertDummiesToReal in comment till the DB will be ready #281

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/451/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
